blog/2024-04-21.md

```diff
@@ -0,0 +1,39 @@
+---
+title: We have a blog now!
+authors: [gagdiez]
+slug: we-have-a-blog
+tags: [updates]
+---
+
+*Check check check. Is this thing on? Hello, world!*
+
+<p><img src="/assets/images/protocol-b73c2a3ace3307226ee7eb2149ee432f.png" /></p>
+
+<!-- truncate -->
+
+## Hello there!
+Welcome to the new NEAR documentation blog! We're excited to inaugurate this new space were we can share news and updates. We know what you are thinking... why a blog? well, because it allows us to interact with you in a different way!
+
+You'll see, when writing documentation it is important for us to keep the content focused and concise. This way, when you are reading it, you have nothing to distract you, and you can focus on learning a new concept.
+
+The problem with this is that many times we wanted to share ideas, thoughts, or insights into why some things are the way they are. However, this would imply going over a tangent, explaining why some decisions where made, which will probably add noise to the document. Most of the times, users just want a link to an example, or working code... also, people don't like to read long texts.
```

Comment:
```suggestion
The problem with this is that many times, we want to share ideas, thoughts, or insights into why some things are the way they are. However, this would imply going off on a tangent and explaining why some decisions were made, which will probably add noise to the document. Most of the time, users just want a link to an example or working code. Also, people don't like to read long texts.
```